README.md

```diff
@@ -173,3 +173,98 @@ rails driver:hmis:dump_graphql_schema
 ```
 
 To pick up the local schema changes on the frontend, run the `graphql:codegen` script (see above).
+
+## Project Structure
+
+This project follows the [bulletproof-react](https://github.com/alan2207/bulletproof-react/blob/master/docs/project-structure.md) structure.
+
+```
+src
+|
++-- api               # graphql queries/mutations/fragments
+|
++-- app               # application layer containing:
+|   +-- App.tsx              # main application component
+|   +-- AppProvider.tsx      # application provider that wraps the entire application
+|   +-- protectedRoutes.ts   # define routes for the authorized applciation
++-- assets            # assets folder can contain all the static files such as images, fonts, etc.
+|
++-- components        # shared components used across the entire application
+|
++-- config            # global configurations, exported env variables etc.
+|
++-- modules           # feature based modules
+|
++-- hooks             # shared hooks used across the entire application
+|
++-- providers         # shared contexts and providers? (added)
```

Comment:
[to discuss tomorrow] Which contexts and providers will live in here? seems like `AppProvider` will move to `app/`, our Theme Provider lives under config (will it stay there?). The remaining apollo client code, I'm not sure if I think of it as a provider? (but maybe this is just a naming thing? :) ) Maybe that should even live under `api/`?